src/main/java/org/springframework/data/elasticsearch/annotations/CompletionContextType.java

```diff
@@ -0,0 +1,21 @@
+package org.springframework.data.elasticsearch.annotations;
+
+/**
+ * Based on reference doc - https://www.elastic.co/guide/en/elasticsearch/reference/current/suggester-context.html
+ *
+ * @author Robert Gruendler
+ */
+public enum CompletionContextType {
+
+	CATEGORY("category"), GEO("geo");
```

Comment:
IMO there is no need to also keep the type as a string. It would be enough to lower the values in the `MappingBuilder`, as we have done with other enums.